Composer/packages/types/src/telemetry.ts

```diff
@@ -227,7 +227,9 @@ type PropertyEditorEvents = {
   RecognizerChanged: { recognizer: string };
 };
 
-type OtherEvents = {};
+type OtherEvents = {
+  TestEvent: undefined;
```

Comment:
Make sure this gets removed

Reply:
This is here for type-correctness (unless we want to do without the "TestEvent" in the unit tests; I could go either way on this).

Reply:
I'm removing TestEvent; I'm just using real event types in the unit tests now.